ejb/WFLY-4699-expose-ejb3-deployment-info.adoc

```diff
@@ -0,0 +1,516 @@
+        ],
+        "delivery-active" => true,
+        "execution-time" => 0L,
+        "invocations" => 0L,
+        "message-destination-link" => undefined,
+        "message-destination-type" => undefined,
+        "messaging-type" => "javax.jms.MessageListener",
+        "methods" => {},
+        "peak-concurrent-invocations" => 0L,
+        "pool-available-count" => 16,
+        "pool-create-count" => 0,
+        "pool-current-size" => 0,
+        "pool-max-size" => 16,
+        "pool-name" => "mdb-strict-max-pool",
+        "pool-remove-count" => 0,
+        "run-as-role" => "Role3",
+        "security-domain" => "other",
+        "timeout-method" => "public void sample.ManagedMDB.timeout(javax.ejb.Timer)",
+        "timers" => [{
+            "time-remaining" => 4213581L,
+            "next-timeout" => 1577768415000L,
+            "calendar-timer" => true,
+            "persistent" => false,
+            "info" => "timer1",
+            "schedule" => {
+                "year" => "*",
+                "month" => "*",
+                "day-of-month" => "*",
+                "day-of-week" => "*",
+                "hour" => "0",
+                "minute" => "0",
+                "second" => "15",
+                "timezone" => undefined,
+                "start" => undefined,
+                "end" => undefined
+            }
+        }],
+        "transaction-type" => "CONTAINER",
+        "wait-time" => 0L,
+        "service" => {"timer-service" => undefined}
+    }
+}
+
+[source]
+/deployment=ejb-management.jar/subsystem=ejb3/message-driven-bean=NoTimerMDB:read-resource(include-runtime)
+{
+    "outcome" => "success",
+    "result" => {
+        "activation-config" => [
+            ("destination" => "java:/queue/NoTimerMDB-queue"),
+            ("destinationType" => "javax.jms.Queue"),
+            ("acknowledgeMode" => "Auto-acknowledge")
+        ],
+        "component-class-name" => "sample.NoTimerMDB",
+        "declared-roles" => [
+            "Role3",
+            "Role2",
+            "Role1"
+        ],
+        "delivery-active" => true,
+        "execution-time" => 0L,
+        "invocations" => 0L,
+        "message-destination-link" => "queue/NoTimerMDB-queue",
+        "message-destination-type" => "javax.jms.Queue",
+        "messaging-type" => "javax.jms.MessageListener",
+        "methods" => {},
+        "peak-concurrent-invocations" => 0L,
+        "pool-available-count" => 16,
+        "pool-create-count" => 0,
+        "pool-current-size" => 0,
+        "pool-max-size" => 16,
+        "pool-name" => "mdb-strict-max-pool",
+        "pool-remove-count" => 0,
+        "run-as-role" => "Role3",
+        "security-domain" => "other",
+        "timeout-method" => undefined,
+        "timers" => [],
+        "transaction-type" => "CONTAINER",
+        "wait-time" => 0L,
+        "service" => undefined
+    }
+}
+
+=== Nice-to-Have Requirements
+
+* ability to read these deployment information from admin console
```

Comment:
I think once they are exposed as runtime resources, they are automatically picked up and displayed in admin console. But if not, yes it can be a follow-up RFE.